clang/test/lab1/safronov_mikhail/test_help_args.cpp

```diff
@@ -0,0 +1 @@
+// RUN: not %clang_cc1 -load %llvmshlibdir/PrintClassesPlugin%pluginext -plugin print-classes --help %s 2>&1 | FileCheck %s
```

Comment:
Here should be at least one CHECK line.

Reply:
I added one check from the last test, it doesn’t work again